wafer_space/projects/tasks.py

```
@@ -898,7 +898,7 @@ def _setup_docker_context(check, project_file, task_instance, logger):
         settings.PRECHECK_TIMEOUT_SECONDS - settings.PRECHECK_SOFT_TIMEOUT_BUFFER
     ),
 )
-def check_project_manufacturability(self, check_id):
+def celery_job_run(self, check_id):  # noqa: PLR0915
```

Ruff (0.14.6)

Unused `noqa` directive (non-enabled: `PLR0915`)

Remove unused `noqa` directive

(RUF100)